src/libraries/Common/src/System/Net/RangeValidationHelpers.cs

```diff
@@ -19,10 +19,7 @@ public static void ValidateSegment(ArraySegment<byte> segment)
             ArgumentNullException.ThrowIfNull(segment.Array, nameof(segment));
 
             // Length zero is explicitly allowed
-            if (segment.Offset < 0 || segment.Count < 0 || segment.Count > (segment.Array.Length - segment.Offset))
-            {
-                throw new ArgumentOutOfRangeException(nameof(segment));
-            }
+            ArgumentOutOfRangeException.ThrowIf(segment.Offset < 0 || segment.Count < 0 || segment.Count > (segment.Array.Length - segment.Offset));
```

Comment:
This no longer includes the argument name.

Reply:
Ah, I see you mentioned that, but it would be good to fix to see what it looks like.

Reply:
Yep, I'll add message (or paramName) arg to `ThrowIf` after regenerating diff. Just not sure should it be simple string or interpolation handler. Mentioned that in linked issue